ios/ReactNativeShareExtension.m

```diff
@@ -196,6 +196,37 @@ - (void)processShareAttachment:(NSEnumerator<NSItemProvider*> *)attachmentEnumer
           @try {
             NSURL *url = (NSURL *)item;
             [dataArray addObject:[url absoluteString]];
+            [self processShareAttachment:attachmentEnumerator intoArray:dataArray withCompletionHandler:handler];
+          } @catch (NSException *exception) {
+            if (handler) {
+              handler(nil, exception);
+            }
+          }
+        }];
+    } else if ([provider hasItemConformingToTypeIdentifier:(NSString *)kUTTypePropertyList]) {
+        [provider loadItemForTypeIdentifier:(NSString *)kUTTypePropertyList options:nil completionHandler:^(NSDictionary *item, NSError *error) {
+
+          @try {
+              NSDictionary *results = (NSDictionary *)item;
```

Comment:
I don't think this cast is necessary, right?  item is already a dict

Reply:
i guess you are right, let me check